src/components/NavBar.vue

```diff
@@ -64,9 +64,11 @@
 
 <script lang="ts">
 import { Component, Prop, Vue } from 'vue-property-decorator';
-import GithubData from '@/services/github-data';
+import { mapGetters } from 'vuex';
 
-@Component
+@Component({
+  computed: mapGetters(['user']),
+})
 export default class NavBar extends Vue {
   private isMobileMenuActive = false;
   private name = '';
```

Comment:
can remove these fields if we're not using them. Also, don't we need one for user?